cypress/integration/payment/create_manual_payment.js

```diff
@@ -155,9 +155,7 @@ describe('Create a manual Payment for a Sales Invoice', function() {
   });
 
   it('Accounting transactions are created', function() {
-    cy.wait(5000); // give the system some breathing space
-    cy.get('body').type('{alt}6'); // open referenced-records-sidelist
-    cy.selectReference('AD_RelationType_ID-540201');
+    cy.openReferencedDocuments('AD_RelationType_ID-540201');
```

Comment:
just a sindenote for next time: we can give internal names to those relation types